src/qt_gui/main_window.cpp

```diff
@@ -158,6 +160,21 @@ void MainWindow::LoadGameLists() {
     }
 }
 
+void MainWindow::GetPhysicalDevices() {
+    if (m_physical_devices.size() == 0) {
```

Comment:
If main window init is called once the size check here is probably unnecessary

Reply:
Forgot to remove that, thanks for pointing it out.
It was some left over of me trying to get this done some other way.